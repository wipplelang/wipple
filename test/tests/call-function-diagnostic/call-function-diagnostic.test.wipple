-- [should error]

produces-text : () -> "Hello, world!"
displays-text : (text :: Text) -> show text

displays-text produces-text
