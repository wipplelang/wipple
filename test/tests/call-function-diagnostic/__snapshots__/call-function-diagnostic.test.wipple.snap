// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`call-function-diagnostic.test.wipple 1`] = `
[
  "/Users/wilson/Desktop/wipple-wasm/test/tests/call-function-diagnostic/call-function-diagnostic.test.wipple:3:16: error: expected pattern here while reading this function inputs",
  "/Users/wilson/Desktop/wipple-wasm/test/tests/call-function-diagnostic/call-function-diagnostic.test.wipple:3:39: error: can't find \`text\`",
]
`;
